packages/sdk/src/types/Vouchers.ts

```diff
@@ -176,56 +431,66 @@ export interface VouchersListParams {
 		after?: string
 		before?: string
 	}
-	order?:
-		| '-created_at'
-		| 'created_at'
-		| '-updated_at'
-		| 'updated_at'
-		| '-type'
-		| 'type'
-		| '-code'
-		| 'code'
-		| '-campaign'
-		| 'campaign'
-		| '-category'
-		| 'category'
+	order?: '-created_at' | 'created_at' | '-updated_at' | 'updated_at' | '-code' | 'code'
 	filters?: {
 		junction?: string
 		[filter_condition: string]: any
 	}
 }
 
 export interface VouchersListResponse {
+	//1_res_vouchers_GET
 	object: 'list'
 	total: number
 	data_ref: 'vouchers'
-	vouchers: VouchersResponse[]
+	vouchers: Omit<VouchersResponse[], 'validation_rules_assignments'>
 }
 
 export type VouchersEnableResponse = VouchersResponse
 
 export type VouchersDisableResponse = VouchersResponse
 
-export interface VouchersImport {
```

Comment:
Interface vs Type - please look at origin comment about the `DistributionsPublicationsCreateParams`